packages/flutter/test/foundation/service_extensions_test.dart

```diff
@@ -104,7 +104,18 @@ void main() {
   test('Service extensions - pretest', () async {
     binding = TestServiceExtensionsBinding();
     expect(binding.frameScheduled, isTrue);
+
+    // We need to test this service extension here because the result should
```

Comment:
nit: maybe change `should always be` to `is`

Reply:
done.